WalletCore/Classes/Core/Extensions/Array+Extensions.swift

```diff
@@ -53,3 +53,14 @@ extension Collection {
         return self.indices.contains(index) ? self[index] : nil
     }
 }
+
+public extension Array where Element:LWSpotWallet {
+    
+    func calculateTotalBalance() -> Decimal {
```

Comment:
Please use meaningful function names as calculateBalanceInBase with the respectively documentation/comments